README.md

```diff
@@ -26,6 +26,14 @@ If you have any other questions not listed here, please raise a PR to have it ad
 
 - Q: How can I __revert__ a commit?
 
+`$ git commit -m "Something terribly misguided"              (1).`
+`$ git reset HEAD~                                           (2).`
```

Comment:
Strictly speaking this is a `reset`. A `reset` is a command to go back in time. a `revert` would be undoing a single commit - I.e. applying a commit *to the end of your commit history* that does the opposite of the specified commit. That is, added lines become deleted, deleted lines are added.